doc_source/EC2GatewayTroubleshooting.md

```diff
@@ -89,7 +89,7 @@ To let AWS Support connect to your gateway, you first log in to the local consol
    You can use the following command to log in to the EC2 instance's local console\. 
 
    ```
-   ssh –i PRIVATE-KEY sguser@INSTANCE-PUBLIC-DNS-NAME
+   ssh –i PRIVATE-KEY admin@INSTANCE-PUBLIC-DNS-NAME
    ```
 **Note**  
```

Comment:
Suggest put the same text just like:

https://docs.aws.amazon.com/storagegateway/latest/userguide/manage-on-premises-common.html

as:

If you created your gateway before June 20th, 2018, please SSH in with:

```
ssh –i PRIVATE-KEY sguser@INSTANCE-PUBLIC-DNS-NAME
```